client/src/catalog-element-readme.html

```diff
@@ -211,6 +211,13 @@
         }
       },
 
+      _remapLinks: function(data, readme) {
+        var githubBase = 'https://github.com/' + data.owner + '/' + data.repo;
+        readme = readme.replace(/(src=")(?!http)\/?([^"]*")/g, '$1' + githubBase + '/raw/master/$2');
+        readme = readme.replace(/(href=")(?!http)\/?([^"]*")/g, '$1' + githubBase + '/blob/master/$2');
```

Comment:
@blasten I don't think we should maintain 2 sets of documentation. Anywhere we want to point to documentation, we should put absolute paths to wc.org.

WDYT?